source/touchHandler.py

```diff
@@ -294,30 +293,57 @@ def notifyInteraction(self, obj):
 
 handler=None
 
-def touchSupported():
+
+def touchSupported(debugLog: bool = False):
 	"""Returns if the system and current NVDA session supports touchscreen interaction.
+	@param debugLog: Whether to log additional details about touch support to the NVDA log.
 	"""
 	if not config.isInstalledCopy() and not config.isAppX:
-		log.debugWarning("Touch only supported on installed copies")
+		if debugLog:
+			log.debugWarning("Touch only supported on installed copies")
 		return False
-	if (winVersion.winVersion.major*10+winVersion.winVersion.minor)<62:
-		log.debugWarning("Touch only supported on Windows 8 and higher")
+	if winVersion.winVersion.platform_version < (6, 2, 9200):

```

Comment:
Is it really necessary to take the platform into account here?

```suggestion
	if (winVersion.winVersion.major, winVersion.winVersion.minor) < (6, 2):
```